README.md

```diff
@@ -1266,6 +1266,33 @@ See [Sophail: Applied attacks against  Antivirus](https://lock.cmpxchg8b.com/sop
 
 Therefore, the best anti-virus is **Common Sense 2017**. See more discussion in [issue #44](https://github.com/drduh/OS-X-Security-and-Privacy-Guide/issues/44).
 
+CylancePROTECT may be worth running for the exploit mitigation features and (when locked down) is much harder to locally bypass than traditional AV, but it's effectiveness at detecting malware on MacOS is questionable.  It's core feature is an algorithm derived from a machine-learning process which aims to identify malware based on various characteristics of a binary executable.  Cylance have a [whitepaper](https://www.cylance.com/content/dam/cylance/pdfs/data_sheets/CylancePROTECT.pdf) with information about how it works.  Single licenses are available from third party resellers such as [Cyberforce](https://cybrforce.com) or [Malware Managed](https://www.malwaremanaged.com).  On MacOS it complements Apple's built-in XProtect by continuously vmmap'ing the memory of active processes to watch for patterns that indicate bad things happening.  At the time of writing it is able to detect and block the following list of events, all of which can be controlled via the Cylance console:
+
+Exploitation:
```

Comment:
This list is too verbose and even I don't know what some of items mean; just remove it altogether, I think.